### ISSUE TITLE
Remote icon URLs don't work with Snarl

### ISSUE THREAD

**Issue by abi** [abi/Yip#4]
Apparently, Snarl doesn't support remote URLs (yet). One of the Snarl guys tells me that it'll be added in the next version. If anyone is inclined to fix this now, you could download icons to a temp file, give that path to Snarl and immediately release them (Snarl makes an own copy so it isn't locked).

**Comment by TlhanGhun** [comment]
Yeah, we'll include remote URLs in the next upcoming Snarl release - long wished feature you might guess ;)

**Comment by abi** [comment]
Great, then Yip doesn't have to be changed at all. I think I'll leave this issue open untill the next Snarl release. Let me know when it's out.
